### PR TITLE
build(deps): bump dependencies

### PR DESCRIPTION
**Pull request by danshilm**
#### Description

Bump dependencies, notably, react-select that has been upgraded to TypeScript.

#### Screenshot (if UI-related)
N/A

#### To-Dos

- [x] Successful build `yarn build`

#### Issues Fixed or Closed

- Fixes none

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -121,22 +120,22 @@
...
-    "eslint-plugin-formatjs": "^2.20.3",
+    "eslint-plugin-formatjs": "^2.20.4",
     "eslint-plugin-jsx-a11y": "^6.5.1",
     "eslint-plugin-prettier": "^4.0.0",
     "eslint-plugin-react": "^7.28.0",
     "eslint-plugin-react-hooks": "^4.3.0",
     "extract-react-intl-messages": "^4.1.1",
     "husky": "^7.0.4",
-    "lint-staged": "^12.2.1",
+    "lint-staged": "^12.2.2",
```
Comment (TheCatLady) [inline package.json:130]: `lint-staged` can be bumped again to `12.3.1`

File: src/styles/globals.css [hunk 2]
```diff
@@ -436,15 +440,6 @@
...
 
-  .hide-scrollbar {
-    -ms-overflow-style: none; /* IE and Edge */
-    scrollbar-width: none; /* Firefox */
-  }
-
-  .hide-scrollbar::-webkit-scrollbar {
-    display: none;
-  }
-
```
Comment (TheCatLady) [inline src/styles/globals.css:447]: Why was this removed?

Comment (danshilm) [reply src/styles/globals.css:447]: Those are duplicates

Comment (TheCatLady) [reply src/styles/globals.css:447]: Ah, I see, good catch!

File: src/components/LanguageSelector/index.tsx [hunk 3]
```diff
@@ -95,7 +91,7 @@ const LanguageSelector: React.FC<LanguageSelectorProps> = ({
   });
 
   return (
-    <Select
+    <Select<OptionType, true>
```
Comment (sct) [inline src/components/LanguageSelector/index.tsx:94]: I think this is the first time I've seen inline generics like this. Wild.

**Review by sct — APPROVED** [review]

File: src/components/LanguageSelector/index.tsx (hunk 3 above)
Comment (danshilm) [reply src/components/LanguageSelector/index.tsx:94]: Same for me, had to take a little time to look at their type definitions because it's pretty wild in there, but also pretty cool
Here's the typings for the `Select` component:

```ts
export class StateManager<
    OptionType extends OptionTypeBase = { label: string; value: string },
    IsMulti extends boolean = false,
    GroupType extends GroupTypeBase<OptionType> = GroupTypeBase<OptionType>,
    T extends SelectBase<OptionType, IsMulti> = SelectBase<OptionType, IsMulti>
> extends Component<
    StateProps<SelectProps<OptionType, IsMulti, GroupType>> &
        Props<OptionType, IsMulti, GroupType> &
        SelectProps<OptionType, IsMulti, GroupType>,
    State<OptionType, IsMulti>
>
```
